components/ee/payment-endpoint/src/accounting/team-subscription-service.ts

```diff
@@ -178,6 +180,7 @@ export class TeamSubscriptionService {
             // Cancel assignee's subscription (if present)
             if (state === "assigned") {
                 const assignedSlot = slot as TeamSubscriptionSlotAssigned;
+                await this.ubpResetOnCancel.resetUsage(assignedSlot.assigneeId);
                 await this.cancelSubscription(db, assignedSlot.assigneeId, ts.planId, slot.id, now);
```

Comment:
Nit: `this.cancelSubscription` already calls `this.ubpResetOnCancel.resetUsage` below.

```suggestion
                await this.cancelSubscription(db, assignedSlot.assigneeId, ts.planId, slot.id, now);
```